### PR TITLE
Build on OpenBSD

### PR DESCRIPTION
**Pull request by klemensn**
The cairo backend works without any platform specific modifications,
just like on Linux.

### CODE REVIEW

**Review by cmyr — APPROVED** [review]
Thanks, this makes sense!

**Comment by klemensn** [comment]
Great, now I'm just waiting for a new proper release so software like `psst` can fetch the crate from the registry rather than a specific git checkout.

This is less important for development on personal machines, but environments such as those for official OS/distribution packages often prefetch things prior to build (which has no network) access, and OpenBSD's package infrastructure only knows how to fetch crates from the registry.

I've been working on fetching from git, which already works, but that also has implications to the build process, which are a bit trickier.

tl;dr: it would be nice to see a new release sooner than later to ease downstream development.

**Comment by cmyr** [comment]
@klemensn no worries, happy to do a release, have been meaning to do the proper 0.5 anyway. :)
